irispy/iris_tools.py

```diff
@@ -282,7 +287,8 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
                 # Calculate baseline SG area for scaling purposes
                 area_sg = iris_response["GEOM_AREA"]
                 for k in range(len(iris_response["INDEX_EL_SG"][nuv, :])):
-                    area_sg = iris_response["GEOM_AREA"] * np.prod(iris_response["INDEX_EL_SG"][nuv, k]["trans"], axis=correct_axis)
+                    #area_sg = iris_response["GEOM_AREA"] * np.prod(iris_response["INDEX_EL_SG"][nuv, k]["trans"], axis=correct_axis)
+                    area_sg = iris_response["GEOM_AREA"] * np.prod(iris_response["ELEMENTS"]["trans"][iris_response["INDEX_EL_SJI"]], axis=1)[nuv, k]
```

Comment:
This should be moved to and replace the original definition [here](https://github.com/sunpy/irispy/pull/108/files/fe1f1daf7d3d4f7d8333a2c9961960d7300a5f64..e43a794df5b3b9b2a30298a2f6f989a6fa3437f7#diff-dee23d5475a6ed5e368938455c92fb2eR288) and the for loop deleted.